extensions/vtk/src/OHIFVTKViewport.js

```diff
@@ -6,12 +6,14 @@ import LoadingIndicator from './LoadingIndicator.js';
 import OHIF from '@ohif/core';
 import PropTypes from 'prop-types';
 import cornerstone from 'cornerstone-core';
-import handleSegmentationStorage from './handleSegmentationStorage.js';
+import cornerstoneTools from 'cornerstone-tools';
 import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
 import vtkImageData from 'vtk.js/Sources/Common/DataModel/ImageData';
 import vtkVolume from 'vtk.js/Sources/Rendering/Core/Volume';
 import vtkVolumeMapper from 'vtk.js/Sources/Rendering/Core/VolumeMapper';
 
+const segmentationModule = cornerstoneTools.getModule('segmentation');
```

Comment:
Need to be careful we're not accidentally creating a hard dependency on `dicom-seg` extension

Reply:
We aren't here, we have a hard dependency on cornnerstone-tools, which is just required to interoperate between `cornerstonejs` and `vtkjs`.

If a `cornerstoneTools` segmentation exists on the displaySet it will be loaded. This could be true whether or not you have a DICOM Seg handler yourself. The segmentation might be created first hand.

Splitting the segmentation handler and the segmentation tools into different extensions solves this problem as you have no possible input to the `cornerstoneTools` segmentation module. If you remove both the vtk plugin will gracefully sweep over the vtk labelmap generation when it simply finds no segmentations in the module.